### PR TITLE
Migrate/community page

### PR DESCRIPTION
**Pull request by bqanderson**
Community page is all set including the use of markdown files for the upcoming events.
This should make updating or adding events, much more simple.

### CODE REVIEW

File: src/pages/blog.js [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
+import { graphql } from 'gatsby'
 import BlogLayout from '../layouts/blog'
 
 import '../styles/main.scss'
 
-export default ({data}) => (
+export default ({ data }) => (
   <BlogLayout>
-    <div className="container">
+    {/* <div className="container">
```
Comment (VinozzZ) [inline src/pages/blog.js:9]: are we gna remove this commented code?

File: src/styles/components/footer.scss [hunk 2]
```diff
@@ -43,6 +43,10 @@ footer {
     }
   }
 
+  input {
```
Comment (VinozzZ) [inline src/styles/components/footer.scss:46]: this is global. Maybe we can put it in with all the other global modifications?

File: src/styles/components/milestones.scss [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+      content: " ";
+      height: 3px;
+      top: auto;
+      bottom: 30px;
+      left: 56px;
+      right: 0;
+      width: 100%;
+      margin-bottom: 20px;
+      margin-left: 8px;
+      background-color: #c4c4c4;
```
Comment (VinozzZ) [inline src/styles/components/milestones.scss]: this could be a variable in _color

File: src/styles/components/milestones.scss [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+        top: auto;
+        bottom: 14px;
+        left: 43px;
+        width: 26px;
+        height: 26px;
+        position: absolute;
+        margin-left: 20px;
+        z-index: 100;
+        border-radius: 50%;
+        background-color: #4f5d7c;
```
Comment (VinozzZ) [inline src/styles/components/milestones.scss]: this could be a variable in _color

File: src/styles/components/milestones.scss [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+          border: 3px solid $white;
+          color: $white;
+        }
+      }
+
+      .milestones-panel {
+        display: inline-block;
+        top: auto;
+        bottom: 64px;
+        left: 0 !important;
```
Comment (VinozzZ) [inline src/styles/components/milestones.scss]: do we need this !important here?

File: src/styles/layouts/comm-layout.scss [hunk 6]
```diff
@@ -1,58 +1,117 @@
...
+    @include media-breakpoint-up(lg) {
+      text-align: left;
+    }
   }
+
+  .btn-forum {
+    padding: 0 $base-height * 7;
+    background-color: $aurora-blue;
+    color: $white;
+    font-family: 'Poppins';
```
Comment (VinozzZ) [inline src/styles/layouts/comm-layout.scss]: I see we have variable for font-family. Maybe we can use those in here

**Review by VinozzZ — CHANGES_REQUESTED** [review]
Overall looks great! I found couple issue with responsiveness of the site and also some issues with font color. The screen shot below shows places where I see the issues, but please double check in the entire site
Screen Shot 2019-09-04 at 5 48 16 PM
Screen Shot 2019-09-04 at 5 43 23 PM
Screen Shot 2019-09-05 at 2 26 18 PM
Screen Shot 2019-09-05 at 2 26 08 PM
Screen Shot 2019-09-05 at 2 25 51 PM
Screen Shot 2019-09-05 at 2 25 40 PM

File: src/pages/blog.js (hunk 1 above)
Comment (bqanderson) [reply src/pages/blog.js:9]: This page has been updated for the blog build out. See PR #247 for further info.

File: src/styles/components/footer.scss (hunk 2 above)
Comment (bqanderson) [reply src/styles/components/footer.scss:46]: I'm not sure this is global. Most of the inputs will have a border, but the email input box on the footer does not.

File: src/styles/components/milestones.scss (hunk 3 above)
Comment (bqanderson) [reply src/styles/components/milestones.scss]: Updated this to match existing gray. `$gray-3` to be precise.

File: src/styles/components/milestones.scss (hunk 4 above)
Comment (bqanderson) [reply src/styles/components/milestones.scss]: Color added to `_colors.scss` as new variable.

File: src/styles/components/milestones.scss (hunk 5 above)
Comment (bqanderson) [reply src/styles/components/milestones.scss]: Actually, I'm not sure we need the `left` or `right` properties at all. I have removed them.

File: src/styles/layouts/comm-layout.scss (hunk 6 above)
Comment (bqanderson) [reply src/styles/layouts/comm-layout.scss]: Yup, update made.

**Pull request closed without merging**